packages/scoped-custom-element-registry/src/scoped-custom-element-registry.ts

```diff
@@ -11,6 +11,53 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
+
+interface CustomElementConstructor {
+  observedAttributes?: Array<string>;
+  formAssociated?: boolean;
+
+  new (...params: unknown[]): CustomHTMLElement;
+}
+
+interface CustomHTMLElement {
+  connectedCallback?(): void;
+  disconnectedCallback?(): void;
+  adoptedCallback?(): void;
+  attributeChangedCallback?(
+    name: string,
+    oldValue?: string | null,
+    newValue?: string | null,
+    namespace?: string | null
+  ): void;
+  formAssociatedCallback?: unknown;
+  formDisabledCallback?: unknown;
+  formResetCallback?: unknown;
+  formStateRestoreCallback?: unknown;
+}
+
+interface CustomElementRegistry {
+  _getDefinition(tagName: string): CustomElementDefinition;
+}
+
+interface CustomElementDefinition {
+  elementClass: CustomElementConstructor;
+  connectedCallback?: CustomHTMLElement['connectedCallback'];
+  disconnectedCallback?: CustomHTMLElement['disconnectedCallback'];
+  adoptedCallback?: CustomHTMLElement['adoptedCallback'];
+  attributeChangedCallback?: CustomHTMLElement['attributeChangedCallback'];
+  formAssociated?: boolean;
+  formAssociatedCallback?: CustomHTMLElement['formAssociatedCallback'];
+  formDisabledCallback?: CustomHTMLElement['formDisabledCallback'];
+  formResetCallback?: CustomHTMLElement['formResetCallback'];
+  formStateRestoreCallback?: CustomHTMLElement['formStateRestoreCallback'];
+  observedAttributes: Set<string>;
+  standInClass?: CustomElementConstructor;
```

Comment:
Add a comment on what standInClass means, and how it's different from elementClass

Reply:
Done